### PR TITLE
Allow coordinate zero values

### PR DESCRIPTION
**Pull request by mschulz-716**
Hello,

I would like to suggest this small change which allows coordinates to have zero values which currently are converted to -1. This only affects values which are exactly 0.

Kind regards,

Matthias

### CODE REVIEW

File: test/gpx.js [hunk 1]
```diff
@@ -2,6 +2,10 @@ import assert from 'assert';
 import fs from 'fs';
 import GPX from '../src/gpx';
 
+if (typeof require !== "undefined") {
+  var DOMParser = require("xmldom").DOMParser;
```
Comment (kf99916) [inline test/gpx.js]: It is not necessary to .be dependent on "xmldom".  Please remove the dependency and revise the new test case (you can refer to "Parse a gpx string" test case).

**Review by kf99916 — CHANGES_REQUESTED** [review]
Hi Matthias,
Thank you for your pull request first. I request changes for your pull request, please help it. Let me know if you have any questions. 🙂

File: test/gpx.js (hunk 1 above)
Comment (mschulz-716) [reply test/gpx.js]: I removed the dependency and changed the test accordingly

**Comment by kf99916** [comment]
Hi Mathias,
Thank you for your work. 👍

**Pull request merged**